lib/v2/screens/transfer/send/send_enter_data/interactor/mappers/send_transaction_mapper.dart

```diff
@@ -23,7 +23,7 @@ class SendTransactionMapper extends StateMapper {
       final selectedFiat = settingsStorage.selectedFiatCurrency;
       final String fiatAmount = currentState.ratesState.fromSeedsToFiat(parsedQuantity, selectedFiat).fiatFormatted;
 
-      eventBus.fire(TransactionSentEventBusEvent(resultResponse.transactionModel));
+      eventBus.fire(const OnNewTransactionEventBus());
 
```

Comment:
Why did you remove the transaction model? Please don't remove that.

Reply:
The idea was if we have the transaction model, we can instantly show it in history as "pending" - this would be cool. So that's why I was putting it in there.

Since we have 500ms block times, it's pretty fast to update, but I think it would add polish down the road to instantly show it.

Reply:
ohh I see now. I'll readd it.

Reply:
re-added